### ISSUE TITLE
cache: failed to create cache table: Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub

### ISSUE THREAD

**Issue by brianlow** [philippta/flyscrape#2]
Any ideas on this?

```
$ wget https://raw.githubusercontent.com/philippta/flyscrape/72f83468299f2510ada7bb2afdc2ca841127fd24/examples/hackernews.js                

$ ./flyscrape run hackernews.js 
cache: failed to create cache table: Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub
```

Macbook M1 using `flyscrape_mac_arm64.tar.gz`. Tried 0.4.0 and 0.3.0 versions

**Comment by billmaya** [comment]
Same here. Download the v0.4.0 and ran from the Mac Mini terminal (M1). 

`$ ./flyscrape dev erasmatazz.js
cache: failed to create cache table: Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub`

**Comment by philippta** [comment]
Flyscrape uses [github.com/mattn/go-sqlite3](https://github.com/mattn/go-sqlite3) for storing the SQLite cache file. This library requires CGO to function correctly which is currently disabled in the release pipeline. That was an oversight.

I tried to enable it but unfortunately it is not trivial. The pipeline must be updated to use something like https://github.com/goreleaser/goreleaser-cross. 

### Workaround
As a workaround, you can install Flyscrape from source:
```
go install github.com/philippta/flyscrape/cmd/flyscrape@latest
```

**Comment by philippta** [comment]
Fixed in the latest [v0.4.1 release](https://github.com/philippta/flyscrape/releases/tag/v0.4.1).

**Comment by brianlow** [comment]
Fixed for me, thanks!
